doc/source/conf.py

```diff
@@ -18,8 +18,14 @@
 
 # These lines added to enable Sphinx to work without installing Ray.
 import mock
-MOCK_MODULES = ["pyarrow",
+MOCK_MODULES = ["gym",
+                "tensorflow",
+                "tensorflow.contrib",
+                "tensorflow.contrib.slim",
+                "tensorflow.contrib.rnn",
+                "pyarrow",
                 "pyarrow.plasma",
+                "smart_open",
```

Comment:
This can be a separate PR, but we should probably modify `setup.py` so people can do `pip install ray[rllib]` to get the relevant dependencies. We can also do `pip install ray[webui]` (see #970).